frontend/src/components/history/MatchList.tsx

```diff
@@ -0,0 +1,60 @@
+import { useEffect, useState } from 'react'
+import { useAppSelector } from '../../store/types'
+import { UserData } from '../../types/UserData'
+import { MatchData } from '../../types/MatchData'
+import styles from './MatchList.module.css'
+
+const MatchList = () => {
+    const [matchHistory, setMatchHistory] = useState([])
+    const userData = useAppSelector((state) => state.user.userData) as UserData
+
+    useEffect(() => {
+        const fetchMatches = async () => {
+            try {
+                const res = await fetch(
+                    `http://localhost:8080/api/match/user/`,
+                    { credentials: 'include' }
+                )
+                const resjson = await res.json()
+                setMatchHistory(resjson)
+                console.log(resjson)
+            } catch (error) {
+                console.log(error)
+            }
+        }
+        fetchMatches()
+    }, [userData.user.id])
+
+    const matchHistoryWinner = matchHistory.map((match: MatchData) => (
+        <div className={styles.player}>
```

Comment:
```suggestion
        <div className={styles.player} key={match.id}>
```
<img width="733" alt="Captura de Pantalla 2023-07-27 a las 12 02 10" src="https://github.com/eprei/ft_transcendence/assets/93380002/a6164e4f-fa6f-489e-91bc-febac1a0fc0c">

To avoid this warning we need to provide a unique "key" prop to each child element generated by the map function. React uses these keys to efficiently update the components in the list and to keep track of their identities.